tools/wpt/testfiles.py

```diff
@@ -252,6 +260,15 @@ def affected_by_wdspec(test):
                     break
         return affected
 
+    def affected_by_interfaces(file_contents):
+        if len(interfaces_changed_names) > 0:
+            if 'idlharness.js' in file_contents:
+                for interface in interfaces_changed_names:
+                    regex = '[\'"]' + interface + '(\\.idl)?[\'"]'
```

Comment:
![image](https://user-images.githubusercontent.com/777956/47220977-8e037c80-d3b3-11e8-8e4d-95dc1a82e2b5.png)

It's the little +/- icon... maybe I'm in an A/B test?

Reply:
I'm actually gonna veto the `r''` suggestion here because I still have to escape the single-quote, and the only gain is the double-backslash of the `\\.idl` which is not very hard to read.

Reply:
I think that @Hexcles might veto your veto, but I'm happy to back down :)

Reply:
Actually, I see the icon too, so I'll use that next time.